src/utils/hooks/useGetFullApplicationStructure.tsx

```diff
@@ -14,14 +14,16 @@ import { IQueryNode } from '@openmsupply/expression-evaluator/lib/types'
 
 interface useGetFullApplicationStructureProps {
   structure: FullStructure
-  shouldProcessValidation?: boolean
+  shouldRevalidate?: boolean
+  revalidateAfterTimestamp?: number
   firstRunValidation?: boolean
 }
 
 const useGetFullApplicationStructure = ({
   structure,
-  shouldProcessValidation,
-  firstRunValidation,
+  shouldRevalidate = false,
+  revalidateAfterTimestamp = 0,
```

Comment:
I'm having trouble understanding this variable from its name. Is there something more specific or precise you could use?

Reply:
hmm if i was to describe it in words: "revalidate after refetch that is later then this timestamp"

we could pass through an object, vs those two to identify that those two settings are grouped (and considered together, not individually), would that be better ?

Reply:
Okay, I get it now. I think my confusion came from the fact that it sounds like a verb ("revalidate..."), yet its value is a number. It's "timestamp to revalidate after". Is calling it `minRefetchTimestamp` any better? Or even `minRefetchTimestampForRevalidation` ? Too verbose? 🤷‍♂️

Reply:
5af4dae0b6a9c6bec320ff3d0db34e392767c079